src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -29,6 +29,12 @@ namespace Microsoft.PowerShell
     [SuppressMessage("Microsoft.Maintainability", "CA1506:AvoidExcessiveClassCoupling")]
     internal partial class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInterface
     {
+
+        /// <summary>
+        /// This is the char that is echoed to the console when the input is masked. This not localizable
+        /// </summary>
+        private const char printToken = '*';
```

Comment:
Style nitpick:  Class field names should begin with underscore character, e.g., '_printToken'.

Reply:
It is const, not field. Our name convention for const is to start with upper case - PrintToken.

Reply:
fixed in 338694dc2a14b5ffa6f24b9bdcd49aea942563e5